aibolit/patterns/if_return_if_detection/if_detection.py

```diff
@@ -24,9 +11,15 @@ class CountIfReturn:
     def __init__(self):
         pass
 
-    def value(self, filename: str):
-        ''''''
-        tree = newJavalangImproved(filename)
-        if_decls = tree.filter([javalang.tree.IfStatement])
-
-        return if_decls
+    def value(self, filename: str) -> List[int]:
+        detected_lines = []
+        p = JavaPackage(filename)
```

Comment:
We shouldn't use single letter variables with large scope or with some complex objects. We better use `java_package` here.

Reply:
got it